_data/tasks.yml

```diff
@@ -23,4 +23,7 @@ toc:
   section:
   - title: Assigning Pods to Nodes
     path: /docs/tasks/administer-cluster/assign-pods-nodes/
-
+- title: Managing Stateful Applications
+  section:
+  - title: Upgrade from Pet Sets to Stateful Sets
```

Comment:
For consistency with other titles, I suggest this title:
Upgrading from PetSets to StatefulSets

If you change the title here, also change it in index.md.